README.md

```diff
@@ -1,91 +1,186 @@
+
+In kuik's namespace, you will find:
+
+- a `Deployment` to run kuik's controllers,
+- a `DaemonSet` to run kuik's image proxy,
+- a `StatefulSet` to run kuik's image cache.
+
+The image cache will obviously require a bit of disk space to run (see [Garbage collection and limitations](#garbage-collection-and-limitations) below). Otherwise, kuik's components are fairly lightweight in terms of compute resources. This shows CPU and RAM usage with the default setup, featuring two controllers in HA mode:
+
+```bash
+$ kubectl top pods
+NAME                                             CPU(cores)   MEMORY(bytes)   
+kube-image-keeper-0                              1m           86Mi            
+kube-image-keeper-controllers-5b5cc9fcc6-bv6cp   1m           16Mi            
+kube-image-keeper-controllers-5b5cc9fcc6-tjl7t   3m           24Mi            
+kube-image-keeper-proxy-54lzk                    1m           19Mi            
+```
 
-When a pod is scheduled, the mutating webhook will rewrite and prefix the image name with `localhost:{port}/` where `port` is configurable. 
-The proxy hostPort setting allows the container runtime to pull images though it on localhost. The proxy will determine if the image should be retrieve either from the local or the source registry.  
+![Architecture](./docs/architecture.jpg)
 
 ## Installation
 
-1. Customize your `values.yaml` to configure the chart.
-1. Install the helm chart following one of the two below methods:
+1. Make sure that you have cert-managed installed. If not, check its [installation page](https://cert-manager.io/docs/installation/) (it's fine to use the `kubectl apply` one-liner, and no further configuration is required).
+1. Install kuik's Helm chart from the [enix/helm-charts](https://github.com/enix/helm-charts) repository:
+
+```bash
+helm upgrade --install \
+     --create-namespace --namespace kuik-system \
+     kube-image-keeper kube-image-keeper \
+     --repo https://charts.enix.io/
+```
+
+That's it!
+
+## Installation with plain YAML files
 
-**From source:**
+You can use Helm to generate plain YAML files and then deploy these YAML files with `kubectl apply` or whatever you want:
 
 ```bash
-helm install --create-namespace --namespace kuik-system kube-image-keeper --values=./values.yaml ./helm/kube-image-keeper/
+helm template --namespace kuik-system \
+     kube-image-keeper kube-image-keeper \
+     --repo https://charts.enix.io/ \
+     > /tmp/kuik.yaml
+kubectl create namespace kuik-system
+kubectl apply -f /tmp/kuik.yaml --namespace kuik-system
 ```
 
-**From [enix/helm-charts](https://github.com/enix/helm-charts) repository:**
+## Configuration and customization
+
+If you want to change e.g. the expiration delay, the port number used by the proxy, enable persistence (with a PVC) for the registry cache... You can do that with standard Helm values.
+
+Check [values.yaml](/helm/kube-image-keeper/values.yaml) for the full list of parameters, their meaning, and default values.
```

Comment:
Maybe we should link the values section from the [artifacthub readme](https://artifacthub.io/packages/helm/enix/kube-image-keeper#values) instead  as it is more readable than yaml and it is generated so it's always up to date. Eventually keep both (values.yaml + artifacthub readme)

Reply:
Good idea; I'll add an extra sentence with a link to the Artifact Hub! 💯